package.json

```diff
@@ -161,6 +161,7 @@
     "@types/ua-parser-js": "^0.7.30",
     "@types/untildify": "^3.0.0",
     "@types/uuid": "^3.4.0",
+    "@types/webdriverio": "^4.13.0",
```

Comment:
What are these types supporting? I don't see an associated `"webdriverio": "<some version>"` above.

Reply:
This is because we've enabled typings for spectron as part of this PR.

Inside `spectron.d.ts` you'll see [this line](https://github.com/electron/spectron/blob/master/lib/spectron.d.ts#L9):

```ts
    import * as WebdriverIO from "webdriverio";
```

Without adding this package, we're not able to leverage spectron's typings.

Reply:
This was specifically added in ccd9a10ef1d8bc4f6a37ec23b5d90e3b37fc7f78 with some context, but if there's another place you'd like to add a comment to emphasise this (can't do comments inside JSON) let me know.

Reply:
My bad, didn't realize there was some context in the description of that commit.